tests/job_timeout_test.py

```diff
@@ -12,7 +12,7 @@
 
 def setup_module(module):
     os.environ.update(
-        SIREPO_JOB_SUPERVISOR_MAX_HOURS_PARALLEL='0.002',
```

Comment:
I'm beginning to think we shouldn't have "MAX_HOURS" and instead be "MAX_SECS". I thought it was better to have the config be this way, but 0.002 is just weird. Thoughts?

Reply:
I agree. I've opened another issue. I'll take care of the Sirepo code and then hand it off to you for rsconf

https://github.com/radiasoft/sirepo/issues/2382